components/match2/wikis/brawlstars/match_group_input_custom.lua

```diff
@@ -58,27 +55,21 @@ function CustomMatchGroupInput.processMap(frame, map)
 	return map
 end
 
--- called from Module:Match/Subobjects
-function CustomMatchGroupInput.processOpponent(frame, opponent)
-	_frame = frame
-	if not Logic.isEmpty(opponent.template) and
-		string.lower(opponent.template) == 'bye' then
-			opponent.name = 'BYE'
-			opponent.type = 'literal'
-	end
+function CustomMatchGroupInput.processOpponent(record, date)
+	local opponent = Opponent.readOpponentArgs(record)
+		or Opponent.empty()
```

Comment:
I don't see this function defined in Module:Opponent?

Reply:
Good catch, should be `Opponent.blank()`